packages/Akvo_rabbitmq_client/src/Akvo_rabbitmq_client/queue_message_util.py

```diff
@@ -0,0 +1,38 @@
+from typing import List, Dict, Optional
+from uuid import uuid4
+
+
+class QueueMessageUtil:
+
+    @staticmethod
+    def create_queue_message(
+        client_id: str,
+        user_id: str,
+        body: str,
+        media: Optional[List[Dict[str, str]]] = None,
+        context: Optional[List[Dict[str, str]]] = None,
+        transformation_log: Optional[List[str]] = None
+    ) -> Dict[str, any]:
+        if media is None:
+            media = []
+        if context is None:
+            context = []
+        if transformation_log is None:
+            transformation_log = [body]
+
+        message = {
+            "conversation_envelope": {
+                "message_id": str(uuid4()),
```

Comment:
The message ID should not be generated server-side, as stated in https://docs.google.com/document/d/12koOL2T5Wstga0EBEwicDZgKq_Y9-dVpOP0OKin5xL8/edit#heading=h.u16r6w9czmp3 I think this is important for deduplication.